### PR TITLE
Criteo destination document

### PR DESCRIPTION
**Pull request by shell10**
Added following files
 - Criteo.md
 - And related screenshots/Images

Please review

### CODE REVIEW

File: help/destinations/catalog/advertising/Criteo.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+# CRITEO DESTINATION
+
+## Overview
+
+_Criteo powers trusted and impactful advertising to bring richer experiences to every consumer across the open internet. With the world's largest commerce data set and best-in-class AI, Criteo ensures each touchpoint across the shopping journey is personalized to reach customers with the right ad, at the right time._
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: Please remove the italicized formatting.

File: help/destinations/catalog/advertising/Criteo.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+>[IMPORTANT]
+>
+>This documentation page was created by Criteo. For any inquiries or update requests, please contact Criteo directly [here](https://marketing.criteo.com/).
+
+
+## Prerequisites
+
+- You need to have an administrator user account on [Criteo Management Center](https://marketing.criteo.com)
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: Please add a full stop after each bullet point.

File: help/destinations/catalog/advertising/Criteo.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+- You'll need your Criteo Advertiser ID (Ask your Criteo contact if you don't have this in hand)
+- Criteo only accepts SHA-256 and plain text emails (to be transformed into SHA-256 before sending). Please do not send any PII (Personal Identifiable Information such as individual's names or phone numbers)
+
+![Prerequisites](../../assets/catalog/advertising/criteo/criteo0.png)
+
+
+This is currently a beta product. Get in touch with your Criteo representative if you would like more details about what this means.
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: Please move this beta information to the top of the page and wrap it in the following tag:

>[!IMPORTANT]
>
>This is currently a beta product. Get in touch with your Criteo representative if you would like more details about what this means.

File: help/destinations/catalog/advertising/Criteo.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+## Supported identities
+
+_Criteo_ supports the activation of identities described in the table below. Learn more about [identities](https://experienceleague.adobe.com/docs/experience-platform/identity/namespaces.html?lang=en#getting-started).
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: Please remove the italic format from Criteo.

File: help/destinations/catalog/advertising/Criteo.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+| Export type | Profile-based | You are exporting all members of a segment, together with the desired schema fields (for example: email address, phone number, last name), as chosen in the select profile attributes screen of the [destination activation workflow](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/activate/activate-batch-profile-destinations.html?lang=en#select-attributes). |
+| Export frequency | Streaming | Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-types.html?lang=en#streaming-destinations). |
+
+## Use cases
+
+To help you better understand how to use Criteo's destination, here are some goals that Adobe Experience Platform customers can achieve with Criteo.
+
+### Use case 1
+
+Get traffic
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: Please format the use case headings like this: `### Use case 1: Get traffic`

File: help/destinations/catalog/advertising/Criteo.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+
+Showcase your business with relevant product offers and flexible creatives. With intelligent product recommendations, your ads will automatically feature the products most likely to trigger visits and engagement. Flexible targeting allows you to build audiences from Criteo's commerce data set or from your own prospect lists and Adobe CDP segments
+
+### Use case 2
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: Same comment as use case 1.

File: help/destinations/catalog/advertising/Criteo.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+
+## Use cases
+
+To help you better understand how to use Criteo's destination, here are some goals that Adobe Experience Platform customers can achieve with Criteo.
+
+### Use case 1
+
+Get traffic
+
+Showcase your business with relevant product offers and flexible creatives. With intelligent product recommendations, your ads will automatically feature the products most likely to trigger visits and engagement. Flexible targeting allows you to build audiences from Criteo's commerce data set or from your own prospect lists and Adobe CDP segments
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: Full stop at the end of the last phrase.

File: help/destinations/catalog/advertising/Criteo.md [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+
+### Use case 2
+
+Increase website conversions
+
+When visitors leave your website, remind them what they're missing with retargeting ads that increase conversions by showing special deals and hyper-relevant offers, wherever they go next. Connect your Adobe CDP segment to re-engage existing customers or target consumers similar to your most loyal shoppers.
+
+## Connect to Criteo
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: Please leave the generic **Connect to the destination** instructions right after the heading (see the template: https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en#connect).

File: help/destinations/catalog/advertising/Criteo.md [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+
+Steps to connect are as follows:
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: Add an H3 heading before this section, called `### Authenticate to Criteo`.

File: help/destinations/catalog/advertising/Criteo.md [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+
+1. Login to Adobe experience platform and connect to Criteo's destination
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: Please update the branding to `Adobe Experience Platform`.

File: help/destinations/catalog/advertising/Criteo.md [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+
+![Auth1](../../assets/catalog/advertising/criteo/criteo2.jpeg)
+
+![Auth2](../../assets/catalog/advertising/criteo/criteo3.jpeg)
+
+![Auth3](../../assets/catalog/advertising/criteo/criteo4.jpeg)
+
+
+
+3. Fill out the details below marked with \*:
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: Please keep the `### Connection parameters `heading from the template for this section.

https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en#parameters

Just below the heading, add the following snippet:

`After authenticating to the destination, please fill in the following connection parameters.`

File: help/destinations/catalog/advertising/Criteo.md [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+   1. Name : A name to help you recognize this destination in the future
+   2. Description : A description to help you identify this destination in the future
+   3. API Version: Preview
+   4. Advertiser ID : Your Criteo advertiser ID
+
+![Details](../../assets/catalog/advertising/criteo/criteo5.jpeg)
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: Please add the relevant screenshot for the connection parameters. The current one does not show the parameter fields.

File: help/destinations/catalog/advertising/Criteo.md [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+
+## Activate segments to this destination
+
+Read [Activate profiles and segments to streaming segment export destinations](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/activate/activate-segment-streaming-destinations.html?lang=en) for instructions on activating audience segments to this destination.
+
+## Exported data
+
+The body of request sent by Adobe Destination Testing API is:
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: The results of the Adobe testing API are not relevant to the end-users who will read this documentation. Instead, let's please add a few lines about how to validate in the Criteo platform that data has successfully come through (e.g.: segments are populating/visible in Criteo).

**Review by vfalon — CHANGES_REQUESTED** [review]
Thank you for submitting this PR! It looks great, just a few minor comments to address before publishing.

File: help/destinations/catalog/advertising/Criteo.md (hunk 1 above)
Comment (shell10) [reply help/destinations/catalog/advertising/Criteo.md]: Done.

File: help/destinations/catalog/advertising/Criteo.md (hunk 2 above)
Comment (shell10) [reply help/destinations/catalog/advertising/Criteo.md]: Done.

File: help/destinations/catalog/advertising/Criteo.md (hunk 4 above)
Comment (shell10) [reply help/destinations/catalog/advertising/Criteo.md]: Done.

File: help/destinations/catalog/advertising/Criteo.md (hunk 5 above)
Comment (shell10) [reply help/destinations/catalog/advertising/Criteo.md]: Done.

File: help/destinations/catalog/advertising/Criteo.md (hunk 10 above)
Comment (shell10) [reply help/destinations/catalog/advertising/Criteo.md]: Done.

File: help/destinations/catalog/advertising/Criteo.md (hunk 3 above)
Comment (shell10) [reply help/destinations/catalog/advertising/Criteo.md]: Done

File: help/destinations/catalog/advertising/Criteo.md (hunk 12 above)
Comment (shell10) [reply help/destinations/catalog/advertising/Criteo.md]: Done

File: help/destinations/catalog/advertising/Criteo.md (hunk 9 above)
Comment (shell10) [reply help/destinations/catalog/advertising/Criteo.md]: Done

File: help/destinations/catalog/advertising/Criteo.md (hunk 7 above)
Comment (shell10) [reply help/destinations/catalog/advertising/Criteo.md]: Done.

File: help/destinations/catalog/advertising/Criteo.md (hunk 13 above)
Comment (shell10) [reply help/destinations/catalog/advertising/Criteo.md]: Done.

File: help/destinations/catalog/advertising/Criteo.md (hunk 11 above)
Comment (shell10) [reply help/destinations/catalog/advertising/Criteo.md]: Done.

File: help/destinations/catalog/advertising/Criteo.md (hunk 8 above)
Comment (shell10) [reply help/destinations/catalog/advertising/Criteo.md]: Done.

File: help/destinations/catalog/advertising/Criteo.md [hunk 14]
```diff
@@ -0,0 +1,128 @@
+> [IMPORTANT]
+> This is currently a beta product. Get in touch with your Criteo representative if you would like more details about what this means.
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: Please move this beta note under the Overview heading.

File: help/destinations/catalog/advertising/Criteo.md [hunk 15]
```diff
@@ -0,0 +1,128 @@
...
+
+# CRITEO DESTINATION
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: Update the title heading to `# Criteo connection`

File: help/destinations/catalog/advertising/Criteo.md [hunk 16]
```diff
@@ -0,0 +1,128 @@
+> [IMPORTANT]
```
Comment (vfalon) [inline help/destinations/catalog/advertising/Criteo.md]: Please rename the file to criteo.md (lowercase c).

**Review by vfalon — CHANGES_REQUESTED** [review]
Thank you for making the requested updates! I have left a few more minor comments before we move forward.

**Pull request merged**